open-banking-accelerator/accelerators/ob-is/carbon-home/dbscripts/open-banking/event-notifications/mssql.sql

```diff
@@ -0,0 +1,43 @@
+CREATE TABLE OB_NOTIFICATION (
```

Comment:
I guess we need to add the license header [1].  Please do the same for the other required places too.

[1] https://github.com/wso2/financial-open-banking/blob/main/open-banking-accelerator/accelerators/ob-is/carbon-home/dbscripts/open-banking/consent/mssql.sql#L1

Reply:
Fixed in https://github.com/wso2/financial-open-banking/pull/33/commits/cde0ef9d3216ab6b595d98a53eb3cd4006300d35